Chapter 03/08_update.py

```diff
@@ -32,4 +32,4 @@
 print("Minimum number of packages of hot dogs required =", number_of_hotdog_packages_needed)
 print("Minimum number of packages of hot dog buns required =", number_of_hotdog_buns_needed)
 print("Number of hot dogs left over =", number_of_hotdogs_left_over)
-print("Number of hot dogs buns left over =", number_of_hotdog_buns_left_over)
```

Comment:
Great, I see what you are saying... Looks good. Next step to get it updated is to change the actual source code on your end and request to merge it to the master...